bug: unable to view application detail page

**Overview**
Application detail page is broken.
Bug appears in both Staging and Production.
Started happening Oct 27 after https://github.com/gitcoinco/grants-round/pull/571 merged?

**Steps To Reproduce**
0. Create a Program and Round if you don't have one already. Create a project on Grant Hub if yo udon't have one already.
1. Apply to the round with a project. (Navigate to the round page on Round Manager, copy link to round application, and visit link to complete application)
2. Refresh the round page on Round Manager. Project should appear in "Received" tab.
3. Click the project card.

**Observed  behavior**
Application detail page fails to load -- see a 404 page.

**Expected behavior**
Expect to see the grant application information (banner, logo, project name, description, application questions/answers, etc)

**Screenshots**

Production:
![Image](https://user-images.githubusercontent.com/62616399/198375611-7663cd7a-7352-491b-8978-c6c6f7c95649.png)

Staging:
![Image](https://user-images.githubusercontent.com/62616399/198375612-cf1e4fdf-6a3e-494c-ac35-717bdff89eff.png)



**Additional context**
Console error:
```
react_devtools_backend.js:4026 error Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transaction={"to":"0x0fF5962Bc56BA0Cf6D7d6EF90df274AE5dC4D16A","data":"0xe63f592c0000000000000000000000000000000000000000000000000000000000000061","accessList":null}, code=CALL_EXCEPTION, version=providers/5.7.0)
    at Logger.makeError (index.ts:269:1)
    at Logger.throwError (index.ts:281:1)
    at fallback-provider.ts:632:1
    at Array.forEach (<anonymous>)
    at FallbackProvider.<anonymous> (fallback-provider.ts:614:1)
    at Generator.next (<anonymous>)
    at fulfilled (fallback-provider.ts:1:1)
```

Getting some failed LIT protocol API calls (some succeed though? this might be normal since not all LIT nodes will have all encryption shares?):
response: `{"errorCode":"encrypted_symmetric_key_not_found","message":"Could not find this encryptedSymmetricKey."}`

payload:
```
{"unifiedAccessControlConditions":[{"contractAddress":"0xd2c7eC130A8b9D99C4620Cd860dCDF6e0bd0CE2F","functionName":"hasRole","functionParams":["0xec61da14b5abbac5c5fda6f1d57642a264ebd5d0674f35852829746dfb8174a5",":userAddress"],"functionAbi":{"name":"hasRole","inputs":[{"name":"role","type":"bytes32"},{"name":"account","type":"address"}],"outputs":[{"name":"","type":"bool"}],"constant":false,"stateMutability":"view"},"chain":"goerli","returnValueTest":{"key":"","comparator":"=","value":"true"}}],"toDecrypt":"55796d4fc32bbaeb7f338524e86c34539e2d45f78417c38fb7721c1b0e267d8e99a31e3347b764e4b09cd3c484c9db05e61e60ef67ab32b63d0783a8a18bd7e04b25a11117e1494ddf660a67f6dbfca4f5134c3b1f62afe3fa9a1abb64d34f8bb5041369ea4e9db6904f8c8881662fdf2320a87a6ef8f3f9ad75e5347037f7cf00000000000000204e2b06558d7e275938322b961854701421b7476e7530912c7b8fc8b2f5039c64abddc201e7a986e17bd06c1b61f936ac","authSig":{"sig":"0x4f33c0532990edae5e193d41888cb09e69f7dee4c5840f89ad5a998bc2c76ed7419ba7883edce59f1166cdca03ee12cb89df6aa5a4cba4fd063a6be09d9984ac1c","derivedVia":"web3.eth.personal.sign","signedMessage":"localhost:3000 wants you to sign in with your Ethereum account:\n0x5bd8C4Ec48cf3EFDD791B5BAb2d4869342605F98\n\n\nURI: http://localhost:3000\nVersion: 1\nChain ID: 5\nNonce: Za7564lIYPAr0PKnK\nIssued At: 2022-08-08T21:10:59.450Z","address":"0x5bd8c4ec48cf3efdd791b5bab2d4869342605f98"},"chain":"goerli"}
```

--- a/packages/round-manager/src/features/api/contracts.ts
+++ b/packages/round-manager/src/features/api/contracts.ts
@@ -18,12 +18,12 @@ export const projectRegistryContract = (
 
   switch (chainId) {
     case ChainId.OPTIMISM_MAINNET_CHAIN_ID: {
-      address = "";
+      address = "0x8e1bD5Da87C14dd8e08F7ecc2aBf9D1d558ea174";
       break;
     }
     case ChainId.GOERLI_CHAIN_ID:
     default: {
-      address = "0x0fF5962Bc56BA0Cf6D7d6EF90df274AE5dC4D16A";
+      address = "0x832c5391dc7931312CbdBc1046669c9c3A4A28d5";
     }
   }
 

